portal-web/test/functional/com/liferay/portalweb/macros/DataEngine.macro

```diff
@@ -1963,7 +1963,7 @@ definition {
 	}
 
 	macro waitForAutoSave {
-		if (IsElementPresent(locator1 = "Button#PUBLISH")) {
+		if (IsElementPresent(key_text = "Publish", locator1 = "Button#ANY")) {
 			Button.clickPublish();
 		}
```

Comment:
The Publish button has changed.
I used in the _**if**_ the same path that was used in the macro `Button.clickPublish()`.
All tests using this macro have been tested.